docs/Remote-Access.md

```diff
@@ -52,35 +52,138 @@ In a more simplified format: SOAP for websites, telnet for command line.
 
 Due to it's ubiquity telnet is easy to use from almost anywhere. 
 
-1. open up a terminal session (or PuTTY) and type in `telnet {{ ip }} {{port}}`
+1. open up a terminal session (or PuTTY) and type in `telnet localhost 3443`
 2. Enter your username and password.
 
 #### Soap
 
-Soap works using POST requests on the HTTP protocol. If you are using php for your server, these is a module that will create and send SOAPs for you, namely:
+Soap works using standard HTTP POST. The entire post payload is XML.
 
+	<SOAP-ENV:Envelope  
```

Comment:
Just for easier understanding of the file when editing, could you add backticks \`\`\`xml around these big codeblocks?

Reply:
it's all good